app.rb

```diff
@@ -10,7 +10,16 @@ class ComplianceViewer < Sinatra::Base
   register Sinatra::ConfigFile
   include ZapReport
 
-  config_file 'credentials.yml'
+  if settings.environment == :test
```

Comment:
would this be better if we required `credentials/test.yml` and `credentials/development.yml`, etc?  That way we could just load yml for the environment.